openml-provider-archetype/src/main/resources/META-INF/maven/archetype.xml

```diff
@@ -0,0 +1,11 @@
+<archetype xmlns="http://maven.apache.org/plugins/maven-archetype-plugin/archetype/1.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/plugins/maven-archetype-plugin/archetype/1.0.0 http://maven.apache.org/xsd/archetype-1.0.0.xsd">
+    <id>openml-provider-archetype</id>
+    <sources>
+        <source>src/main/java/openml/MyOpenmlProvider.java</source>
+        <source>src/main/java/openml/MyOpenmlModelLoader.java</source>
+        <source>src/main/java/openml/MyOpenmlModel.java</source>
+        <source>src/main/java/openml/MyFirstOpenmlModel.java</source>
+        <source>src/main/java/openml/MySecondOpenmlModel.java</source>
```

Comment:
change the name of those classes from \*Open**ml**\* to \*Open**ML**\*

Reply:
I don't really agree: MyOpenMLModel is not as readable as MyOpenmlModel. But of course, it's not a big change, can do it either way the team agrees with